src/harberger-lib/components/Web3Connect.res

```diff
@@ -39,34 +37,9 @@ let make = (~clickAction=() => ()) => {
         ReasonReactRouter.push("#")
         connectWeb3(RootProviderTypes.NoAction)
       }}>
-      {"Connect"->React.string}
+      {"Log In"->React.string}
```

Comment:
My friend had no idea what "connect" meant. She thought it was like "connect with the team/support/help" or something else.

Reply:
On the fence with this one slightly. I am happy with Log in for now if you like but to our current user base 'Connect' is more relevant. Happy for it to change as further down the line with Torus login it will be a login type thing.

Reply:
Yeah, agree, I guess we need to try reduce confusion for a broader audience.

Also, if they click anywhere to do a transaction for example it will prompt them to connect anyway.